GVFS/GVFS.Common/HeartbeatThread.cs

```diff
@@ -53,6 +53,7 @@ private void EmitHeartbeat(object unusedState)
                 EventLevel eventLevel = EventLevel.Verbose;
                 EventMetadata metadata = this.dataProvider.GetMetadataForHeartBeat(ref eventLevel) ?? new EventMetadata();
                 DateTime now = DateTime.Now;
+                metadata.Add("Version", ProcessHelper.GetCurrentProcessVersion());

```

Comment:
Would we be better off calling `ProcessHelper.GetCurrentProcessVersion()` once and caching the result?  I'm not sure how expensive that call it.